docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+services:
+  postgres:
+    image: postgres:latest
+    container_name: postgres-db
+    environment:
+      POSTGRES_DB: lms
+      POSTGRES_USER: user
+      POSTGRES_PASSWORD: password
+    ports:
+      - 5432:5432
+    volumes:
+      - ./postgres_data:/var/lib/postgresql/data
```

Comment:
it's better to named volume to do not store data on local machine (for local development it's fine, but on production it should be avoided)